initramfs-tools/scripts/init-premount/tcg_opal

```diff
@@ -28,10 +28,17 @@ list_block_devices()
 	fi
 }
 
-is_opal_drive()
+opal_drive_locked_status()
```

Comment:
Can you please move this function inside `is_locked_opal_drive`? Should make the code leaner.